source/databricks/tests/test_time_series_transformer.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import pandas as pd
+from package.schemas import time_series_unprocessed_schema
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    StringType,
+    IntegerType,
+    TimestampType,
+    LongType,
+)
+from datetime import datetime
+from decimal import Decimal
+from package.transforms.time_series_transformer import (
+    transform_unprocessed_time_series_to_points,
+)
+
+
+@pytest.fixture(scope="module")
+def time_series_unprocessed_factory(spark, timestamp):
+    def factory(
+        CreatedDateTime: TimestampType() = timestamp("2022-06-09T12:09:15.000Z"),
+        RegistrationDateTime: TimestampType() = timestamp("2022-06-10T12:09:15.000Z"),
+        StartDateTime: TimestampType() = timestamp("2022-06-08T12:09:15.000Z"),
+        Resolution: LongType() = 2,
```

Comment:
Python naming standards

```suggestion
        created_date_time: TimestampType() = timestamp("2022-06-09T12:09:15.000Z"),
        registration_date_time: TimestampType() = timestamp("2022-06-10T12:09:15.000Z"),
        start_date_time: TimestampType() = timestamp("2022-06-08T12:09:15.000Z"),
        resolution: LongType() = 2,
```